live-examples/js-examples/intl/intl-relativetimeformat-prototype-resolvedoptions.html

```diff
@@ -0,0 +1,15 @@
+<pre>
+<code id="static-js">var rtf1 = new Intl.RelativeTimeFormat('de-DE');
+var options1 = rtf1.resolvedOptions();
+
+console.log(options1.locale);
+// expected output (Firefox / Safari): "de-DE"
+// expected output (Chrome): "de"
+
+console.log(options1.style);
+// expected output: "long"
+
+console.log(options1.numeric);
+// expected output: "always"
+</code>
```

Comment:
Do you think it would be worth creating a couple of objects, with different options, to emphasise the connection between the options passed in and the result of this function? For example:

```
var rtf1 = new Intl.RelativeTimeFormat('en', { style: 'narrow' });
var options1 = rtf1.resolvedOptions();

var rtf2 = new Intl.RelativeTimeFormat('es', { numeric: 'auto' });
var options2 = rtf2.resolvedOptions();

console.log(`${options1.locale}, ${options1.style}, ${options1.numeric}`);
// expected output: "en, narrow, always"

console.log(`${options2.locale}, ${options2.style}, ${options2.numeric}`);
// expected output: "es, long, auto"
```
?